_data/api.yml

```diff
@@ -1748,7 +1748,7 @@ sections:
               }
             }
       "delete an address":
-        description: Delete all addresses associated with a given tag, optionally associated with a pool
+        description: Delete addresses associated with a given tag: either all addresses, those associated with a specified pool, or a single address.
```

Comment:
the code seem to support specifying a pool and address.  is it wrong or is this line?